src/map/clif_packetdb.hpp

```diff
@@ -2327,18 +2327,22 @@
 	packet(0x0A37,59);
 #endif
 
-// 2016-09-28cRagexeRE
-#if PACKETVER >= 20160928
-	parseable_packet(0x0A97,8,clif_parse_dull,0);
-	parseable_packet(0x0A99,4,clif_parse_dull,0);
-	parseable_packet(0x0A9C,2,clif_parse_dull,0);
-#endif
-
 // 2016-10-26bRagexeRE
 #if PACKETVER >= 20161026
 	packet(0x0AA5,-1);
 #endif
 
+// 2017-02-08bRagexeRE
+#if PACKETVER >= 20170208
+	parseable_packet(0x0A97,8,clif_parse_equipswitch_add,2,4);
+	packet(0x0A98,10);
+	parseable_packet(0x0A99,4,clif_parse_equipswitch_remove,2,4);
```

Comment:
Trying to unequip boots:
`[Warning] clif_parse: Received unsupported packet (packet 0x0040, 4 bytes received), disconnecting session #3.`

0x0A99 have size 8 on client 20170215.

Reply:
Sorry I do not have that client to confirm this. :(

Reply:
I can upload here. I will open a repo of clients unpacked by myself soon.
[2017-02-15aRagexeRE](https://www.mediafire.com/file/9sfvlf8u02u0hrl/) - MediaFire

Reply:
This has been fixed as well. :)